pkgs/tools/networking/yggdrasil/default.nix

```diff
@@ -28,10 +28,11 @@ buildGoModule rec {
   '';
 
   meta = with lib; {
-    description = "An experiment in scalable routing as an encrypted IPv6 overlay network";
+    description =
```

Comment:
not sure why this was changed

Reply:
`nixfmt` wanted to reduce the line length